app/api/healthcareAuthorities/getHealthcareAuthoritiesApi.ts

```diff
@@ -1,5 +1,6 @@
 import Yaml from 'js-yaml';
-import { AUTHORITIES_LIST_URL_MVP1 } from '../../constants/authorities';
+import env from 'react-native-config';
```

Comment:
minor nitpick, but what do you think about a line to break the imports from the const?

Reply:
agree!